src/main/kotlin/no/nav/syfo/behandlendeenhet/kafka/BehandlendeEnhetProducer.kt

```diff
@@ -0,0 +1,39 @@
+package no.nav.syfo.behandlendeenhet.kafka
+
+import no.nav.syfo.behandlendeenhet.database.domain.PPerson
+import org.apache.kafka.clients.producer.KafkaProducer
+import org.apache.kafka.clients.producer.ProducerRecord
+import org.slf4j.LoggerFactory
+
+class BehandlendeEnhetProducer(
+    private val kafkaProducerBehandlendeEnhet: KafkaProducer<String, KBehandlendeEnhetUpdate>,
+) {
+    fun updateBehandlendeEnhet(
+        pPerson: PPerson,
+    ) {
+        try {
+            kafkaProducerBehandlendeEnhet.send(
+                ProducerRecord(
+                    BEHANDLENDE_ENHET_UPDATE_TOPIC,
+                    pPerson.personident,
```

Comment:
Mener forresten å huske at vi byttet til å bruke UUID (generert fra personident) i stedet for personident som key (i hvert fall i https://github.com/navikt/isoppfolgingstilfelle og https://github.com/navikt/isdialogmotekandidat/). For å unngå at personident havner i loggene f.eks. hvis konsumenter logger key.

Se f.eks. https://github.com/navikt/isdialogmotekandidat/pull/45

Reply:
Aha, det har jeg ikke vært borti før, den var fin!🏅